src/components/ConfirmationBox/ConfirmationBoxSelectors.js

```diff
@@ -1,6 +1,6 @@
 export const getPredictionName = state => {
 	return state.storeList && state.prediction
-		? state.storeList[state.prediction.id].name
+		? state.storeList.find(item => item.id === state.prediction.id).name
```

Comment:
Was it mentioned in the demo that "unknown" is now a guess?

If so would this fail since unknown is not in the store list(?)

Reply:
Unknown is ignored before it gets to this stage, it's handled by the model internally. The prediction should only ever be a valid store ID.